.github/workflows/release.yaml

```
@@ -0,0 +1,40 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      image_name:
+        description: The image name in the format <username>/<image>.
+        required: true
+        default: splatform/mits
+
+env:
+  IMAGE_NAME: "${{ github.event.inputs.image_name }}"
+  IMAGE_FILTER_REFERENCE: "${{ github.event.inputs.image_name }}"
+  REPOSITORY: "${{ github.repository }}"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+        fetch-depth: 0
+    - name: Create new tag
+      run: ./build/ci/create_next_tag.sh
+    - name: Build image
+      run: ./build/build.sh
+    - name: Image registry login
+      run: |-
+        docker login \
+          --username "${{ secrets.DOCKER_USERNAME }}" \
+          --password-stdin <<<"${{ secrets.DOCKER_PASSWORD }}"
```

```suggestion
      run: >-
        docker login
          --username "${{ secrets.DOCKER_USERNAME }}"
          --password-stdin <<<"${{ secrets.DOCKER_PASSWORD }}"
```

Or, even better, use the [action dedicated for it](https://github.com/docker/login-action):
```suggestion
      uses: docker/login-action@v1
      with:
        username: ${{ secrets.DOCKER_USERNAME }}
        password: ${{ secrets.DOCKER_PASSWORD }}
```

---

Nice!